src/slider/slider.component.ts

```diff
@@ -179,6 +191,44 @@ export class Slider implements AfterViewInit, ControlValueAccessor {
 	get value() {
 		return this._value;
 	}
+
+	/** Set the initial value. Available for two way binding */
+	@Input() set value2(v) {
```

Comment:
We should be able to use an array to represent the high and low/left and right values ... That way we aren't specifically coding in `value1` and `value2` ... In theory we should be able to support `N` sliders, but we can start with just a tuple.

So `value` should have the type `number | number[]` ... As long as we eimt a new array each time `ngModel` should work fine with this setup. I would expect that we swap everything internally to use arrays, and just dump the contents of `this.value[0]`. `isRange` can probably just be a check for `this.value.length > 1` 🤔

Reply:
I had similar thoughts.